services/QuillLMS/app/models/cron.rb

```diff
@@ -28,6 +28,16 @@ def self.interval_1_day
     Question::TYPES.each {|type| RefreshQuestionCacheWorker.perform_async(type) }
   end
 
+  # Configured in Heroku Scheduler to run at XX:20
+  def self.run_hourly_at_20_minute_mark
+    ResetGhostInspectorAccountWorker.perform_async
+  end
+
+  # Configured in Heroku Scheduler to run at XX:50
+  def self.run_hourly_at_50_minute_mark
+    ResetGhostInspectorAccountWorker.perform_async
+  end
+
```

Comment:
Note that Heroku Scheduler is configured to trigger rake tasks, so I think you'll need to add corresponding tasks to `lib/tasks/cron.rake` if you want these to be run on that timer.  An alternative might be to add code to the "ten minute" interval that calls these if certain conditions are true (the way we trigger `run_saturday_ inside `interval_1_day`).  Not sure which approach is better.

Reply:
Thank you!  I was banging my head on this yesterday since the scheduler was reporting that they were run:
![Screen Shot 2022-02-01 at 8 08 52 AM](https://user-images.githubusercontent.com/2057805/151974170-41b7fc20-61d1-4724-917e-60c1bb52e63e.png)

yet I wasn't seeing anything happening.